src/_templates/build_pages.sh

```diff
@@ -0,0 +1,12 @@
+# create pages
```

Comment:
This is a nice idea but I tend to avoid these, since using the rm is highly risky and I already broke my whole os installation by running a similar file. 
Also the built version is on a different branch so this won't automate the whole process. So I would rather drop this unless you can do this and move the built files to the gh-pages branch automatically.